### PR TITLE
Pull request

### PR DESCRIPTION
**Pull request by sethhinckley**
Sorry if the branch name "pull request" is confusing

### CODE REVIEW

File: src/edu/pcc/cis/conversion/ConversionFunctionsTest.java [hunk 1]
```diff
@@ -100,5 +100,11 @@ public void invalidMonthReturnsEmptyString() {
         assertEquals("", ConversionFunctions.monthNumToMonthName(30));
     }
 
-    // Adding this comment to make a nominal change for pull request. Merged too soon.
+    // SH: Adding this comment to make a nominal change for pull request. Merged too soon.
+    // SH: Adding a test for non-integer numbers so I can meet the lab minimum of 2 commits.
+    // This should result in a data type error.
+    @Test
+    public void invalidNumberReturnsError() {
+        assertEquals("", ConversionFunctions.monthNumToMonthName(6.5));
```
Comment (caratang) [inline src/edu/pcc/cis/conversion/ConversionFunctionsTest.java]: This won't compile unless you change the argument of the method-under-test to double (which shouldn't be done).

**Comment by caratang** [comment]
The GitHub project was supposed to be the hello-world project. I don't want solutions to the unit test project in public repos on GitHub. Would you mind removing this repository or making it private?
